gridded_etl_tools/utils/zarr_methods.py

```diff
@@ -289,17 +323,25 @@ def parallel_subprocess_files(
         commands = []
         for existing_file in input_files:
             new_file = existing_file.with_suffix(replacement_suffix)
-            commands.append(  # map will convert the file names to strings because some command line tools (e.g. gdal) don't like Pathlib objects
-                    list(map(str, command_text + [existing_file, new_file]))
-                 )
+            if invert_file_order:
+                commands.append(  # map will convert the file names to strings because some command line tools (e.g. gdal) don't like Pathlib objects
+                        list(map(str, command_text + [new_file, existing_file]))
```

Comment:
A bit minor, but `[new_file, existing_file]` should be stored in a variable to avoid copying the rest of the code.